guides/common/attributes-satellite.adoc

```diff
@@ -68,8 +68,12 @@
 :foreman-installer-package: satellite-installer
 :foreman-installer: satellite-installer
 :foreman-maintain: satellite-maintain
-:FreeIPA: Red{nbsp}Hat Identity Management
+:FreeIPA: Red{nbsp}Hat Identity{nbsp}Management
```

Comment:
With https://github.com/theforeman/foreman-documentation/pull/3015/commits/9611bd36eb0c4a808fc12ff896592012862f9aee, this PR now uses only attr "FreeIPA". I decided not to use "IdM" as an abbreviation anywhere mostly because there aren't that many occurrences (so spelling the d/s product name out should not give anyone too big a headache), but also because it's probably safer to use precise naming in non-FreeIPA docs.

Let me know if there are any concerns. This affects RH d/s only; no change upstream.